package.json

```diff
@@ -391,7 +391,7 @@
     "@codemirror/gutter": "^0.19.9",
     "@codemirror/highlight": "^0.19.8",
     "@codemirror/history": "^0.19.2",
-    "@codemirror/language": "^0.19.10",
+    "@codemirror/language": "0.19.10",
```

Comment:
just curious why drop the ^ ?

Reply:
good q! I'll revert that, was accidental. I churned the versions a bit - turns out they need to be synced and none can be upgraded to 0.20 until all of them can be.